.vscode/extensions.json

```diff
@@ -0,0 +1,10 @@
+{
+  "recommendations": [
+    "dbaeumer.vscode-eslint",
```

Comment:
Nitpick: Based on the conventional commit message, there are plugins unrelated to stylelint. Maybe the message can be more about setting up recommended plugins for VSCode?